redbot/core/commands/commands.py

```diff
@@ -809,6 +820,100 @@ def help(self):
         if doc:
             return inspect.cleandoc(translator(doc))
 
+    async def red_get_data_for_user(self, *, user_id: int) -> MutableMapping[str, io.BytesIO]:
+        """
+        This should be overridden by cogs
+        that support users requesting their data
+        as well as cogs that don't store data.
+
+        Overriden implementations should send a io.BytesIO containing a
```

Comment:
Should this say "send" or "return"?
Also, it just says it needs to return the `io.BytesIO`, but the function states it returns a `MutableMapping[str, io.BytesIO]`. This should probably be explained in this more human-readable section.